src/components/UI/organisms/MenuItem/MenuItem.css

```diff
@@ -0,0 +1,114 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap');
+
+:root {
```

Comment:
That root shouldn't be in this file, besides Mateusz is using `css modules` why you don't?